### PR TITLE
[Dualtor] Skip `test_ecmp_sai_value` on dualtor

### PR DESCRIPTION
**Pull request by lolyu**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [x] 202311
- [x] 202405

#### What is the motivation for this PR?
As the subject.

Signed-off-by: Longxiang Lyu <lolv@microsoft.com>

#### How did you do it?
If the topo is dualtor, skip it.

#### How did you verify/test it?
Run over dualtor testbed.

### CODE REVIEW

**Review by ZhaohuiS — APPROVED** [review]
LGTM

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -527,6 +527,7 @@ ecmp/test_ecmp_sai_value.py:
     conditions_logical_operator: or
     conditions:
       - "topo_type not in ['t1', 't0']"
+      - "'dualtor' in topo_name"
```
Comment (yutongzhang-microsoft) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:530]: LGTM

**Review by yutongzhang-microsoft — APPROVED** [review]

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 2]
```diff
@@ -527,6 +527,7 @@ ecmp/test_ecmp_sai_value.py:
...
       - "asic_type not in ['broadcom']"
```
Comment (yutongzhang-microsoft) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:531]: Also this condition, we have `pytest.mark.asic('broadcom')` in script

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 3]
```diff
@@ -527,6 +527,7 @@ ecmp/test_ecmp_sai_value.py:
     conditions_logical_operator: or
     conditions:
       - "topo_type not in ['t1', 't0']"
```
[the remark this answers is not in the document]
Comment (lolyu) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:529]: that's out of the scope for this PR

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml (hunk 2 above)
Comment (lolyu) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:531]: Same as the above.

**Pull request closed without merging**